src/main/java/io/jenkins/plugins/opentelemetry/computer/MonitoringComputerListener.java

```diff
@@ -81,6 +79,33 @@ public void postConstruct() {
                 .setDescription("Number of agents")
                 .setUnit("1")
                 .build();
+        Jenkins jenkins = Jenkins.getInstanceOrNull();
+        if (jenkins != null) {
+            QuickDiskUsagePlugin diskUsagePlugin = jenkins.getPlugin(QuickDiskUsagePlugin.class);
```

Comment:
if the plugin is not installed I think this will cause a class not found exception

Reply:
Thanks for asking, I also was skeptical about whether it will work.

For the record, I tested it out in my local environment:

1) Run a docker jenkins intance with this opentelemetry version
2) Run some jobs
3) Install the disk plugin
4) Run some more jobs

```
2021-04-30 15:12:13.955+0000 [id=182]	INFO	o.j.p.workflow.job.WorkflowRun#finish: it/log #1 completed: SUCCESS
2021-04-30 15:12:58.251+0000 [id=182]	INFO	o.j.p.workflow.job.WorkflowRun#finish: it/log #2 completed: SUCCESS
2021-04-30 15:16:38.460+0000 [id=250]	INFO	o.j.p.workflow.job.WorkflowRun#finish: it/log #3 completed: SUCCESS
^[[A2021-04-30 15:17:06.931+0000 [id=11]	INFO	hudson.PluginManager#install: Starting installation of a batch of 1 plugins plus their dependencies
2021-04-30 15:17:07.381+0000 [id=261]	INFO	h.model.UpdateCenter$DownloadJob#run: Starting the installation of cloudbees-disk-usage-simple on behalf of anonymous
2021-04-30 15:17:08.364+0000 [id=261]	INFO	h.m.UpdateCenter$UpdateCenterConfiguration#download: Downloading cloudbees-disk-usage-simple
2021-04-30 15:17:08.407+0000 [id=261]	INFO	h.model.UpdateCenter$DownloadJob#run: Installation successful: cloudbees-disk-usage-simple
2021-04-30 15:17:08.407+0000 [id=261]	INFO	h.m.UpdateCenter$CompleteBatchJob#run: Completing installing of plugin batch…
2021-04-30 15:17:08.408+0000 [id=261]	INFO	hudson.ClassicPluginStrategy#updateDependency: Updated dependency of opentelemetry
2021-04-30 15:17:08.643+0000 [id=268]	INFO	jenkins.InitReactorRunner$1#onAttained: Started initialization
2021-04-30 15:17:08.644+0000 [id=272]	INFO	jenkins.InitReactorRunner$1#onAttained: Listed all plugins
2021-04-30 15:17:08.646+0000 [id=278]	INFO	jenkins.InitReactorRunner$1#onAttained: Prepared all plugins
2021-04-30 15:17:08.655+0000 [id=282]	INFO	jenkins.InitReactorRunner$1#onAttained: Started all plugins
2021-04-30 15:17:08.655+0000 [id=270]	INFO	jenkins.InitReactorRunner$1#onAttained: Augmented all extensions
2021-04-30 15:17:08.656+0000 [id=273]	INFO	jenkins.InitReactorRunner$1#onAttained: System config loaded
2021-04-30 15:17:08.656+0000 [id=277]	INFO	jenkins.InitReactorRunner$1#onAttained: System config adapted
2021-04-30 15:17:08.656+0000 [id=277]	INFO	jenkins.InitReactorRunner$1#onAttained: Loaded all jobs
2021-04-30 15:17:08.657+0000 [id=280]	INFO	jenkins.InitReactorRunner$1#onAttained: Configuration for all jobs updated
2021-04-30 15:17:08.657+0000 [id=284]	INFO	c.c.s.QuickDiskUsagePlugin$2#run: Re-estimating disk usage
2021-04-30 15:17:08.658+0000 [id=277]	INFO	jenkins.InitReactorRunner$1#onAttained: Completed initialization
2021-04-30 15:17:08.660+0000 [id=261]	INFO	h.m.UpdateCenter$CompleteBatchJob#run: Completed installation of 1 plugins in 1.7 sec
2021-04-30 15:17:09.979+0000 [id=284]	INFO	c.c.s.QuickDiskUsagePlugin$2#run: Finished re-estimating disk usage.
2021-04-30 15:17:24.027+0000 [id=255]	INFO	o.j.p.workflow.job.WorkflowRun#finish: it/log #4 completed: SUCCESS
2021-04-30 15:21:28.389+0000 [id=315]	INFO	hudson.model.AsyncPeriodicWork#lambda$doRun$0: Started Periodic background build discarder
2021-04-30 15:21:28.620+0000 [id=315]	INFO	hudson.model.AsyncPeriodicWork#lambda$doRun$0: Finished Periodic background build discarder. 231 ms

```

So the metrics were generated firstly without the new metric (jenkins.disk.size) and then after the installation of the disk plugin those metrics were sent correctly.

![image](https://user-images.githubusercontent.com/2871786/116717924-57ba0580-a9d1-11eb-94bd-4a52a6072b46.png)

![image](https://user-images.githubusercontent.com/2871786/116717827-3c4efa80-a9d1-11eb-96b8-b02855df83a8.png)